extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -1557,33 +1550,34 @@ private SonosZoneGroup getCurrentZoneGroup() {
 
     /**
      * Sets the volume level for a notification sound
-     * (initializes {@link #notificationSoundVolume})
      *
-     * @param command
+     * @param notificationSoundVolume
      */
-    public void setNotificationSoundVolume(Command command) {
-        if (command != null) {
-            notificationSoundVolume = command.toString();
+    public void setNotificationSoundVolume(PercentType notificationSoundVolume) {
+        if (notificationSoundVolume != null) {
+            // set new volume only if it is higher than the current volume
```

Comment:
Why? My notification sounds are usually static files that volume range is known.
The different internet radio could require a louder volume level, so why not using the notification volume that has been defined and adding such a logic here?

Reply:
I can tell you. I want to preserve a little bit of the previous functionality. The `notificationVolume` property covers maybe 90% of your notifications and will be used every time. You cannot change the value during runtime in any case. And then there is this one situation when you need a very high noise (e.g. an intruder alert). With this condition you are able to set the normal volume in a rule to the abnormal level and it prevents the binding from using your default notification volume.